pkgs/development/python-modules/aiohttp-apispec/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder
+, aiohttp, webargs, apispec3, jinja2
+}:
+
+buildPythonPackage rec {
+  pname = "aiohttp-apispec";
+  version = "2.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hhlmh3mc3xg68znsxyhypb5k12vg59yf72qkyw6ahg8zy3qfz2m";
+  };
+
+  disabled = pythonOlder "3.0";
+
+  propagatedBuildInputs = [ aiohttp webargs apispec3 jinja2 ];
+
+  doCheck = false;
```

Comment:
Please add a comment why tests are disabled and add a pythonImportsCheck.

Reply:
I think you just need to add pytestCheckHook to checkInputs and maybe ``tests`` to pytestFlagsArray

Reply:
I tried the hook thing before, no success. 0 tests.
I will try the pytestFlagsArray.

Reply:
The tests are not included in the archive file from pypi.

Reply:
noted

Reply:
Can we fetch form source repository instead?